helmcharts/notes/values.yaml

```diff
@@ -0,0 +1,127 @@
+  fullnameOverride: ""
+
+  service:
+    type: ClusterIP
+    port: 80
+
+  ingress:
+    enabled: false
+    annotations: {}
+      # kubernetes.io/ingress.class: nginx
+      # kubernetes.io/tls-acme: "true"
+    hosts:
+      - host: notes.local
+        paths: []
+
+    tls: []
+    #  - secretName: chart-example-tls
+    #    hosts:
+    #      - chart-example.local
+
+  resources: {}
+    # We usually recommend not to specify default resources and to leave this as a conscious
+    # choice for the user. This also increases chances charts run on environments with little
+    # resources, such as Minikube. If you do want to specify resources, uncomment the following
+    # lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  # limits:
+  #   cpu: 100m
+  #   memory: 1024Mi
+  # requests:
+  #   cpu: 100m
+  #   memory: 1024Mi
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+
+  extraInitContainers: []
+
+  # config is exposed to the app container on disk as a secret
+  # these values are exposed to the app code via django_settings
+  config: 
+    ALLOWED_HOSTS:
+    - '*'
+     # should match those configured for `edx-notes` Client in EdX's /admin/oauth2/client/
+    CLIENT_ID: ''
+    CLIENT_SECRET: ''
+    DATABASES:
+      default:
+        ENGINE: django.db.backends.mysql
+        HOST: notes-mysql
+        NAME: notes-db
+        OPTIONS:
+          connect_timeout: 10
+        PASSWORD: ''
+        PORT: '3306'
+        USER: notes-db-user
+    DISABLE_TOKEN_CHECK: false
+    ELASTICSEARCH_INDEX: edx_notes
+    ELASTICSEARCH_URL: http://notes-elasticsearch-client:9200
+    HAYSTACK_CONNECTIONS:
+      default:
+        ENGINE: notesserver.highlight.ElasticsearchSearchEngine
+        INDEX_NAME: notes
+        URL: http://notes-elasticsearch-client:9200/
+    JWT_AUTH:
+      JWT_AUTH_COOKIE_HEADER_PAYLOAD: stage-edx-jwt-cookie-header-payload
+      JWT_AUTH_COOKIE_SIGNATURE: stage-edx-jwt-cookie-signature
+      JWT_AUTH_REFRESH_COOKIE: stage-edx-jwt-refresh-cookie
+      JWT_ISSUERS: []
+      JWT_PUBLIC_SIGNING_JWK_SET: ''
+    RESULTS_DEFAULT_SIZE: 25
+    RESULTS_MAX_SIZE: 250
+    SECRET_KEY: ''
+    USERNAME_REPLACEMENT_WORKER: username_replacement_service_worker
+    LOG_SETTINGS_LOG_DIR: '/var/tmp'
+    LOG_SETTINGS_LOGGING_ENV: 'no_env'
+    LOG_SETTINGS_DEV_ENV: True
+    LOG_SETTINGS_DEBUG: True
+    LOG_SETTINGS_LOCAL_LOGLEVEL: 'INFO'
+    LOG_SETTINGS_EDX_FILENAME: "edx.log"
+    LOG_SETTINGS_SERVICE_VARIANT: 'edx-notes-api'
+
+# See the Elasticsearch chart provider's website in requirements.yaml for a full list of possible overrides
+elasticsearch:
+  enabled: true
+
+  image:
+    repository: "elasticsearch"
+    tag: "1.5.2"
+    pullPolicy: "IfNotPresent"
+
+# See the Mysql chart provider's website in requirements.yaml for a full list of possible overrides
+mysql:
+  enabled: true
+  
+  image: mysql
+  imageTag: 5.6.45
+  existingSecret: notes-mysql-secrets
```

Comment:
I have so much to learn :guitar:

Reply:
I added a comment explaining this because it was actually kind of hard to sort through and I feel like it's unclear now that you called it out the comment reads:

`  # We create the secret ourselves in templates/secrets.yaml
  # Defining the name of the secret in templates/_helpers.tpl
  # and use the existing secret directive to tell the mysql chart not to create
  # secrets to use ours instead.
  # this is done so we can use the secret from our values.yaml since mysql and 
  # the django application need to have the same DB password.
  # This is a common practice in the helm stable charts 
  # https://github.com/helm/charts/tree/master/stable
  # as these charts are often used to compose higher level charts.
  # but should be able to run on their own out of the box.`